fix: change plugin-web-search to use google.serper.dev

eksion opened the pull request:
<!-- Use this template by filling in information and copying and pasting relevant items out of the HTML comments. -->

# Relates to

<!-- LINK TO ISSUE OR TICKET -->

<!-- This risks section must be filled out before the final review and merge. -->

# Risks

<!--
Low, medium, large. List what kind of risks and what could be affected.
-->

# Background

## What does this PR do?

## What kind of change is this?

<!--
Bug fixes (non-breaking change which fixes an issue)
Improvements (misc. changes to existing features)
Features (non-breaking change which adds functionality)
Updates (new versions of included code)
-->

<!-- This "Why" section is most relevant if there are no linked issues explaining why. If there is a related issue, it might make sense to skip this why section. -->
<!--
## Why are we doing this? Any context or related work?
-->

# Documentation changes needed?

<!--
My changes do not require a change to the project documentation.
My changes require a change to the project documentation.
If documentation change is needed: I have updated the documentation accordingly.
-->

<!-- Please show how you tested the PR. This will really help if the PR needs to be retested and probably help the PR get merged quicker. -->

# Testing

## Where should a reviewer start?

## Detailed testing steps

<!--
None: Automated tests are acceptable.
-->

<!--
- As [anon/admin], go to [link]
  - [do action]
  - verify [result]
-->

<!-- If there is a UI change, please include before and after screenshots or videos. This will speed up PRs being merged. It is extra nice to annotate screenshots with arrows or boxes pointing out the differences. -->
<!--
## Screenshots
### Before
### After
-->

<!-- If there is anything about the deployment, please make a note. -->
<!--
# Deploy Notes
-->

<!--  Copy and paste command line output. -->
<!--
## Database changes
-->

<!--  Please specify deploy instructions if there is something more than the automated steps. -->
<!--
## Deployment instructions
-->

<!-- If you are on Discord, please join https://discord.gg/ai16z and state your Discord username here for the contributor role and join us in #development-feed -->
<!--
## Discord username

-->

odilitime reviewed (changes requested):
Either needs to provide an option to change behavior of plugin-web-search or be a separate plugin completely. I do not want to merge PRs that remove functionality

renovate[bot] committed:
Update pnpm to v9.15.0 [SECURITY]

viv-cheung committed:
Add more data when scrapping tweets (#2644)

Co-authored-by: Sayo <hi@sayo.wtf>
Co-authored-by: tcm390 <60634884+tcm390@users.noreply.github.com>

AIFlowML committed:
fixed all the typing errors and ommissio. leaved the memories variable maybe Ninja want to use it later

AIFlowML committed:
Merge branch 'develop' into fix-plugin-quick-intel

odilitime committed:
Merge pull request #2888 from elizaOS/renovate/npm-pnpm-vulnerability

Update pnpm to v9.15.0 [SECURITY]

odilitime committed:
Merge pull request #2887 from elizaOS/renovate/npm-systeminformation-vulnerability

Update dependency systeminformation to v5.23.8 [SECURITY]

odilitime committed:
Merge pull request #2879 from v1xingyue/plugin-sui-patch

fet: use axios to fetch price and support two types private key

AIFlowML committed:
Fixed all the typing and also the reassign. Leaved 4 reassign because i could not understand the logic (#2902)

Co-authored-by: Shakker Nerd <165377636+shakkernerd@users.noreply.github.com>

AIFlowML committed:
Fixed the state reassignment (#2899)

Co-authored-by: Shakker Nerd <165377636+shakkernerd@users.noreply.github.com>

AIFlowML committed:
fix: plugin-opacity (#2900)

* Fixed all the typing errors

* chore: remove console.log

---------

Co-authored-by: Shakker Nerd <165377636+shakkernerd@users.noreply.github.com>

AIFlowML committed:
fixed all the typing errors. most of the actions had also a bad structur or missing code from the action.ts (#2898)

Co-authored-by: Shakker Nerd <165377636+shakkernerd@users.noreply.github.com>

wtfsayo committed:
patch via pnpm (#2901)

Co-authored-by: Shakker Nerd <165377636+shakkernerd@users.noreply.github.com>

AIFlowML committed:
fixed all the typing issues (#2893)

Co-authored-by: Shakker Nerd <165377636+shakkernerd@users.noreply.github.com>

AIFlowML committed:
fix: plugin-quai  (#2892)

* fixed all logical and typing issue

* feat: use elizaLogger

---------

Co-authored-by: Shakker Nerd <165377636+shakkernerd@users.noreply.github.com>

shakkernerd committed:
Merge branch 'develop' into fix-plugin-quick-intel

shakkernerd committed:
Merge pull request #2890 from AIFlowML/fix-plugin-quick-intel

fix: plugin-quick-intel

shakkernerd committed:
Merge branch 'develop' into fix-plugin-sei-v1

shakkernerd committed:
feat: use elizaLogger

shakkernerd committed:
Merge pull request #2877 from AIFlowML/fix-plugin-sei-v1

fix: plugin-sei

shakkernerd committed:
Merge branch 'develop' into fix-plugin-sgx

shakkernerd committed:
Merge pull request #2872 from AIFlowML/fix-plugin-sgx

fix: plugin-sgx

shakkernerd committed:
Merge branch 'develop' into fix-plugin-starknet

shakkernerd committed:
Merge pull request #2866 from AIFlowML/fix-plugin-starknet

fix: plugin-starknet

shakkernerd committed:
Merge branch 'develop' into fix-plugin-spheron

shakkernerd committed:
Merge pull request #2870 from AIFlowML/fix-plugin-spheron

fix: plugin-spheron

AIFlowML committed:
Fixed some logic and typing (#2868)

Co-authored-by: Shakker Nerd <165377636+shakkernerd@users.noreply.github.com>

maximevtush committed:
Update LICENSE (#2903)

wtfsayo committed:
Merge branch 'develop' into tests/plugin-asterai

AIFlowML committed:
Fix partially all the typings. The code is a not follow any logic so i clean more i could (#2905)

madschristensen99 committed:
Add files via upload

Pricstas committed:
chore: fix spell errors (#2909)

* fix typo README.md

* fix typos createToken.ts

* fix typo submitVoteAction.ts

* fix typo README.md

* fix typo README.md

* fix typo README.md

madschristensen99 committed:
Update index.ts

eric2hen committed:
feat: new model provider for LM Studio

madschristensen99 committed:
Update README.md

madschristensen99 committed:
Update README.md

mmarfinetz committed:
feat(arbitrage): improve gas price optimization and volume calculation (#2869)

* feat(arbitrage): improve gas price optimization and volume calculation

* use elizaLogger

* Update pnpm-lock.yaml

---------

Co-authored-by: 0xmarf <mitch@integral.link>
Co-authored-by: Sayo <hi@sayo.wtf>

wtfsayo committed:
Merge branch 'develop' into feature/lmstudio-model-provider

ai16z-demirix committed:
Merge branch 'develop' into tests/plugin-asterai

shakkernerd committed:
Merge pull request #2860 from ai16z-demirix/tests/plugin-asterai

chore(add-tests): plugin-asterai -> test coverage and config

shakkernerd committed:
chore: add version to package.json

shakkernerd committed:
chore: pnpm lock file

shakkernerd committed:
v0.1.8-alpha.1

madschristensen99 committed:
Merge branch 'elizaOS:main' into main

madschristensen99 committed:
Update package.json

madschristensen99 committed:
Update package.json

shakkernerd committed:
ci: changes to allow package scoping to public

shakkernerd committed:
Merge pull request #2928 from elizaOS/fix/lerna

ci: changes to allow package scoping to public

shakkernerd committed:
fix: missing version prop in package.json

shakkernerd committed:
Merge pull request #2929 from elizaOS/fix/plugin-avail

fix: missing version prop in package.json

shakkernerd committed:
fix: invalid npm package naming convention

shakkernerd committed:
Merge pull request #2930 from elizaOS/fix/plugin-lens-network

fix: invalid npm package naming convention

shakkernerd committed:
chore: inconsistent folder naming conventipn

shakkernerd committed:
Merge pull request #2931 from elizaOS/fix/plugin-lens-network

chore: inconsistent folder naming conventipn

michavie committed:
fix template typos

shakkernerd committed:
feat: add public access to packages.

This ensures all packages that are not marked private are properly scoped and can be published to npm registry.

shakkernerd committed:
Merge pull request #2933 from elizaOS/feat/ci

feat: add public access to packages.

shakkernerd committed:
Merge branch 'develop' into fix/template-typos

shakkernerd committed:
Merge pull request #2932 from michavie/fix/template-typos

fix template typos

shakkernerd committed:
Merge branch 'main' into develop

shakkernerd committed:
Merge branch 'develop' into feature/lmstudio-model-provider

shakkernerd committed:
Merge pull request #2913 from eric2hen/feature/lmstudio-model-provider

feat: new model provider for LM Studio

AIFlowML committed:
Fixed 50% of the typing, To fix all we need to a rewrite the code logic too

AIFlowML committed:
Merge branch 'develop' into fix-plugin-nft-generation

tcm390 committed:
remove duplicated plugins

tcm390 committed:
add missing dependencies

tcm390 committed:
resolve type conflict caused by multiple versions of @solana/web3.js

shakkernerd committed:
Merge pull request #2934 from AIFlowML/fix-plugin-nft-generation

fix: plugin-nft-generation

AIFlowML committed:
Fixed all the typing and the errors

AIFlowML committed:
Merge branch 'develop' into fix-plugin-nft-collection

tcm390 committed:
update dependencies

AIFlowML committed:
Fixed all the typing and literal errors

shakkernerd committed:
Merge branch 'develop' into tcm-fix-build

AIFlowML committed:
Merge branch 'develop' into fix-plugin-news

shakkernerd committed:
Merge pull request #2938 from elizaOS/tcm-fix-build

fix: building error

shakkernerd committed:
Merge branch 'develop' into fix-plugin-news

shakkernerd committed:
Merge pull request #2940 from AIFlowML/fix-plugin-news

fix: plugin-news

shakkernerd committed:
Merge branch 'develop' into fix-plugin-nft-collection

shakkernerd committed:
Merge pull request #2937 from AIFlowML/fix-plugin-nft-collection

fix: plugin-nft-collection

AIFlowML committed:
Fixed all the errors and typing

AIFlowML committed:
Fixed all the errors and typing

AIFlowML committed:
Merge branch 'develop' into fix-plugin-multiversx

AIFlowML committed:
Fixed all the errors

tcm390 committed:
Merge branch 'develop' into fix-plugin-movement

tcm390 committed:
Merge pull request #2941 from AIFlowML/fix-plugin-near

fix: plugin-near

tcm390 committed:
Merge branch 'develop' into fix-plugin-movement

tcm390 committed:
Merge pull request #2943 from AIFlowML/fix-plugin-movement

fix: plugin-movement

tcm390 committed:
clean code

tcm390 committed:
Merge pull request #2942 from AIFlowML/fix-plugin-multiversx

fix: plugin-multiversx

tcm390 committed:
Merge pull request #2945 from elizaOS/tcm-clean-code

fix: clean code

tcm390 committed:
correct plugin path

tcm390 committed:
add missing plugins back

tcm390 committed:
Merge pull request #2947 from elizaOS/tcm-missing-plugins

fix: add missing plugins in package.json

tcm390 committed:
add missing maxTweetLength state

tcm390 committed:
Merge branch 'develop' into tcm-add-missing-state

AIFlowML committed:
Fixed all the errors

AIFlowML committed:
Merge branch 'develop' into fix-plugin-moralis

tcm390 committed:
Merge pull request #2951 from elizaOS/tcm-add-missing-state

fix: {{maxTweetLength}} doesn't work in tweet post template

wtfsayo committed:
Merge branch 'develop' into fix-plugin-moralis

AIFlowML committed:
Added only logging for the context

AIFlowML committed:
Merge branch 'develop' into fix-mind-network

AIFlowML committed:
Fixed all the errors

AIFlowML committed:
Merge branch 'develop' into fix-plugin-mina

AIFlowML committed:
Fixed all the errors.

AIFlowML committed:
Merge branch 'develop' into fix-plugin-massa

tcm390 committed:
Merge pull request #2955 from AIFlowML/fix-plugin-massa

fix: plugin-massa

tcm390 committed:
Merge pull request #2954 from AIFlowML/fix-plugin-mina

fix-plugin-mina

tcm390 committed:
Merge pull request #2953 from AIFlowML/fix-mind-network

fix: mind-network

tcm390 committed:
Merge pull request #2952 from AIFlowML/fix-plugin-moralis

fix: plugin-moralis

wonseokjung committed:
fix: remove unnecessary @ts-expect-error directive in chat component (#2950)

* fix: remove unnecessary @ts-expect-error directive in chat component

* update pnpm version!

---------

Co-authored-by: Sayo <hi@sayo.wtf>

btspoony committed:
feat(plugin-di): add `Dependency Injection` support for services and clients, and di plugin samples to _examples as new folder (#2855)

* feat: let services and client injectable

* feat: add example for plugin with di

* fix: add missing service binding

* docs: Enhance Dependency Injection README with usage examples and clarifications

* docs: Update DI README with refined service and provider injection descriptions

* docs: Refine DI plugin README with clearer class and usage descriptions

---------

Co-authored-by: Sayo <hi@sayo.wtf>

tcm390 committed:
handle whitespace in quote conversion

tcm390 committed:
add more test case

tcm390 committed:
Merge branch 'develop' into tcm-fix-array-parsing

tcm390 committed:
Merge pull request #2961 from elizaOS/tcm-fix-array-parsing

fix: handle whitespace in quote conversion

AIFlowML committed:
Fixed all the errors (#2960)

Co-authored-by: Sayo <hi@sayo.wtf>

AIFlowML committed:
Fixed all erros also missing zod (#2959)

Co-authored-by: Sayo <hi@sayo.wtf>

AIFlowML committed:
fixed all the errors. only 3 any remain because not opssible to test (#2957)

Co-authored-by: Sayo <hi@sayo.wtf>

YohanGH committed:
Update of the complete translation of README_EN.md

- Add all sections
- Link repair img break

AIFlowML committed:
Fixed typing and missing parameters. Fixed also the reassignment logic (#2906)

Co-authored-by: Sayo <hi@sayo.wtf>

tcm390 committed:
ensure action tweet replies to agent's initial tweet

AIFlowML committed:
fix: plugin-router-nitro (#2884)

* fixed all the logic error and added proper typing

* refactor code

* Update executeSwap.ts

---------

Co-authored-by: Sayo <hi@sayo.wtf>

tcm390 committed:
disable prettier

tcm390 committed:
Merge branch 'develop' into tcm-twitter-reply-chain

tcm390 committed:
Merge pull request #2966 from elizaOS/tcm-twitter-reply-chain

fix: ensure action tweet replies to agent's initial tweet

AIFlowML committed:
fix: plugin-lensnetwork  (#2965)

* Fixed all the errors and also the Zod schema

* Other fix

* Merge remote-tracking branch 'origin/develop' into pr/2965

* Update executeSwap.ts

---------

Co-authored-by: Sayo <hi@sayo.wtf>

wtfsayo committed:
bump pnpm for remaining workflows (#2968)

YohanGH committed:
Fix error in extra space line 117 and translate Here in Ici on line 182

YohanGH committed:
Fix error in extra space line 129 and typing erro line 131

AIFlowML committed:
fixed al the type issues (#2971)

AIFlowML committed:
Fixed all the issues (#2970)

Co-authored-by: Sayo <hi@sayo.wtf>

AIFlowML committed:
fix: plugin-irys (#2969)

* Fixed all the errors and also the Zod schema

* Fixed all errors in a bit invasive way. I saved a _bk copy that i keep as reference.

---------

Co-authored-by: Sayo <hi@sayo.wtf>

AIFlowML committed:
Fixed all issues but leave 2 any that deterioiate all the other files if changed (#2973)

Co-authored-by: Sayo <hi@sayo.wtf>

AIFlowML committed:
Fixed all the typing issues (#2974)

Co-authored-by: Sayo <hi@sayo.wtf>

AIFlowML committed:
Fixed all the typing issues (#2976)

Co-authored-by: Sayo <hi@sayo.wtf>

AIFlowML committed:
Fixed imports. Reassign had to leave. (#2978)

Co-authored-by: Sayo <hi@sayo.wtf>

ai16z-demirix committed:
feat(chore): plugin-arbitrage test config and coverage (#2983)

* plugin-arbitrage: vitest config

* plugin-arbitrage: arbitrage action tests

* plugin-arbitrage: Arbitrage test

* plugin-arbitrage: index test

* plugin-arbitrage: arbitrage service test

* plugin-arbitrage: test setup

* plugin-arbitrage: test market setup

* plugin-arbitrage: test script

madschristensen99 committed:
Merge branch 'develop' into main

derrix060 committed:
feat: improve instagram client (#2975)

* fix(instagram): get post interval setting from the correct place

There is a INSTAGRAM_ setting that should be respected first, then
look into the generic one, then the default value

* feat(instagram): enhance image generation settings

Updated the image generation parameters to utilize dynamic settings
from the runtime configuration. This allows for more flexible image
generation based on user-defined preferences, improving the overall
functionality and customization of the Instagram post service.

* test(instagram): add comprehensive configuration and post service tests

Expanded test coverage for Instagram client configuration and post service. Added tests for:
- Environment configuration validation with enhanced image settings
- Handling of partial and invalid configuration parameters
- Image generation settings and fallback mechanisms
- Post interval selection logic
- Error handling for image generation failures

---------

Co-authored-by: Sayo <hi@sayo.wtf>

wtfsayo committed:
Merge branch 'develop' into main

madschristensen99 committed:
Update package.json

madschristensen99 committed:
Update index.ts

madschristensen99 committed:
Update package.json

ai16z-demirix committed:
feat(chore): plugin-apro -> test coverage and test config (#2980)

* plugin-apro: test config

* plugin-apro: attps tests

* plugin-apro: create and register agent tests

* plugin-apro: verify data tests

* plugin-apro: index tests

---------

Co-authored-by: Sayo <hi@sayo.wtf>

wtfsayo committed:
Merge branch 'develop' into main

madschristensen99 committed:
Update package.json

elpulpo0 committed:
fix README's CONTRIBUTING.md link in plugin-multiversx package (#2985)

Co-authored-by: Sayo <hi@sayo.wtf>

yaruno committed:
docs: fix broken links to github (#2987)

* fix links to github

* Update actions.md

* Update actions.md

---------

Co-authored-by: Sayo <hi@sayo.wtf>

wtfsayo committed:
Merge branch 'develop' into main

Dahka2321 committed:
Update lpmanager.character.json (#2921)

futreall committed:
chore: correction typos chat_2024-12-04.md (#2986)

* Update chat_2024-12-04.md

* Update chat_2024-12-04.md

wtfsayo committed:
Merge branch 'develop' into main

ai16z-demirix committed:
plugin-autonome: test script

ai16z-demirix committed:
plugin-autonome: test config

ai16z-demirix committed:
plugin-autonome: actions tests

ai16z-demirix committed:
plugin-autonome: actions tests - package json

odilitime committed:
Merge pull request #2912 from madschristensen99/main

feat(more-actions): Lit Protocol plugin

odilitime committed:
re-enable wtfsayonara's patch

ai16z-demirix committed:
Merge branch 'develop' into tests/plugin-autonome

odilitime committed:
re-enable wtfsayonara's patch (#2993)

ai16z-demirix committed:
Merge branch 'develop' into tests/plugin-autonome

odilitime committed:
audit bump

odilitime committed:
chore: lockfile audit (#2994)

* re-enable wtfsayonara's patch

* audit bump

ai16z-demirix committed:
Merge branch 'develop' into tests/plugin-autonome

ai16z-demirix committed:
plugin-avail: package json scripts

ai16z-demirix committed:
plugin-avail: vitest config

ai16z-demirix committed:
plugin-avail: submit data tests

ai16z-demirix committed:
plugin-avail: transfer tests

ai16z-demirix committed:
Merge branch 'develop' into tests/plugin-avail

ai16z-demirix committed:
plugin-avalanche: package json

ai16z-demirix committed:
plugin-avalanche: vitest config

ai16z-demirix committed:
plugin-avalanche: token mil create tests

shakkernerd committed:
Merge branch 'develop' into patch-1

shakkernerd committed:
fix: start script link

shakkernerd committed:
Merge pull request #2964 from YohanGH/patch-1

Update of the complete translation of README_FR.md [FR]

shakkernerd committed:
Merge branch 'develop' into tests/plugin-avalanche

shakkernerd committed:
Merge pull request #2999 from ai16z-demirix/tests/plugin-avalanche

feat (chore): plugin avalanche test config and coverage

shakkernerd committed:
Merge branch 'develop' into tests/plugin-avail

shakkernerd committed:
Merge pull request #2997 from ai16z-demirix/tests/plugin-avail

feat (chore): plugin avail test config and coverage

shakkernerd committed:
Merge branch 'develop' into tests/plugin-autonome

shakkernerd committed:
Merge pull request #2992 from ai16z-demirix/tests/plugin-autonome

feat (chore): plugin-autonome test config and coverage

ileana-pr committed:
fix OpenAI embedding issue

shakkernerd committed:
Merge pull request #3003 from ileana-pr/develop

fix: OpenAI embedding issue

odilitime committed:
Merge branch 'develop' of https://github.com/elizaos/eliza into odi-fix

maxime committed:
Ensure the connection between user and room before creating a memory, fixing the constraint error in PostgreSQL

maxime committed:
Merge branch 'develop' into bug/client-slack-and-postgres

odilitime committed:
don't start services twice, improving logging with which agent

eksion committed:
OKOK

odilitime committed:
bump lock

tcm390 committed:
add normalizwed function

tcm390 committed:
remove unnecessary regex

tcm390 committed:
revert

odilitime committed:
Merge pull request #3007 from elizaOS/odi-fix

fix: don't start services twice, improve logging

AIFlowML committed:
Fixed all the errors and the reassign of state

AIFlowML committed:
Merge branch 'develop' into fix-plugin-hyperliquid

tcm390 committed:
null check

AIFlowML committed:
Fixed all the typing errors, all the logical error, refactored the action

AIFlowML committed:
Merge branch 'develop' into fix-plugin-holdstation

wtfsayo committed:
Merge branch 'develop' into tcm-normalize-json-string

tcm390 committed:
Merge branch 'tcm-normalize-json-string' into tcm-transcribe-null-check

tcm390 committed:
Merge pull request #3014 from elizaOS/tcm-transcribe-null-check

fix: deepgram transcription null check

tcm390 committed:
Merge pull request #3008 from elizaOS/tcm-normalize-json-string

fix: llava model parsing Issue

tcm390 committed:
Merge branch 'develop' into bug/client-slack-and-postgres

AIFlowML committed:
fixex all the types error and the blocking error in the GoPlusManage

AIFlowML committed:
Merge branch 'develop' into fix-plugin-goplus

eksion committed:
jajaja

tcm390 committed:
Merge branch 'develop' into fix-plugin-hyperliquid

tcm390 committed:
Merge pull request #3011 from AIFlowML/fix-plugin-hyperliquid

fix(lint): plugin-hyperliquid

tcm390 committed:
Merge branch 'develop' into fix-plugin-holdstation

tcm390 committed:
Merge pull request #3015 from AIFlowML/fix-plugin-holdstation

fix(lint):  plugin-holdstation

tcm390 committed:
Merge branch 'develop' into fix-plugin-goplus

tcm390 committed:
Merge pull request #3016 from AIFlowML/fix-plugin-goplus

fix(lint): plugin-goplus

0xSero committed:
chore(lint-BIOME): Improving errors and warnings (#2990)

* clean up 15 errors and 60 warnings

* smoke

* smokeTest pass remove track

* set up lockfile

---------

Co-authored-by: Sero <69639595+Seroxdesign@users.noreply.github.com>
Co-authored-by: Sayo <hi@sayo.wtf>

wtfsayo committed:
Merge branch 'develop' into bug/client-slack-and-postgres

tcm390 committed:
Merge pull request #3006 from maxime/bug/client-slack-and-postgres

fix: client-slack & adapter-postgres: Ensure the connection between user and room before creating a memory,…

eksion committed:
ok

eksion committed:
Merge remote-tracking branch 'upstream/develop' into SRISE

eksion committed:
news puller

eksion committed:
asdasd

eksion committed:
asdas

eksion committed:
xx

eksion committed:
tpl

eksion committed:
xx

eksion committed:
xxx

eksion committed:
lower the treshold

eksion committed:
restore it to 0.85

eksion committed:
improveee

eksion committed:
image descriptions

eksion committed:
fix

odilitime commented:
Hello,

We are changing our plugin development strategy to be more scalable. We have moved the plugins out into their own repos and we're looking for people to either maintain those or own them on their own Github.

Please make your code changes to this repo: https://github.com/elizaos-plugins/plugin-web-search

If you'd like to be a maintainer,  file an issue in the plugin repo and join our Discord https://discord.gg/elizaos to coordinate.

If you'd like to control the plugin on your own Github, please add an issue to the plugin repo pointing to your repo, and add a modification to the registry. Submit a PR to edit the registry here: https://github.com/elizaos-plugins/registry

Closing this PR for now. Let us know if you have any questions.

eksion committed:
added neobera character

eksion committed:
limit post

eksion committed:
change neocortex to neo knowledge

eksion committed:
updated neobera character

eksion committed:
dual

eksion committed:
another fix

eksion committed:
asdasd

eksion committed:
testing

eksion committed:
refactor multi client

eksion committed:
aaa

eksion committed:
asdasd

eksion committed:
xxx

eksion committed:
fix try

eksion committed:
fix try

eksion committed:
OKOK

eksion committed:
asdasd

eksion committed:
improving prompt

eksion committed:
update neobera

eksion committed:
update system prompt bera

eksion committed:
fucking emojis

eksion committed:
fix images

eksion committed:
backup
